CadRevealComposer/ModelMetadata.cs

```diff
@@ -1,38 +1,33 @@
 ﻿namespace CadRevealComposer;
 
-using System;
 using System.Collections.Generic;
-using System.Text.Json.Serialization;
-using System.Linq;
-using System.Text;
-using System.Threading.Tasks;
 using System.Text.Json;
 
 public class ModelMetadata
 {
-    Dictionary<string, string> metadata;
+    readonly Dictionary<string, string> _metadata;
 
-    public ModelMetadata(Dictionary<string, string> _metadata)
+    public ModelMetadata(Dictionary<string, string> metadata)
     {
-        metadata = _metadata;
+        this._metadata = metadata;
     }
 
     public int Count()
     {
-        return this.metadata.Count;
+        return this._metadata.Count;
     }
 
     public void Add(ModelMetadata modelMetadata)
     {
-        foreach (var kvp in modelMetadata.metadata)
+        foreach (var kvp in modelMetadata._metadata)
```

Comment:
Wait, I just filled a hole in my c# knowledge. The private access modifier is class based not object based 🤯

Reply:
YEEEEP